index.html

```diff
@@ -101,12 +100,12 @@ <h2 class="tonic">Tonic</h2>
               
               <a href="/someurl" class="sec3rlink3  ">JAVASCRIPT</a>
           </li>
+        </ul>
 
+          <button type="submit" class="sec3rbutton sec3rlink4 ">See Projects</button>
+    

```

Comment:
Let's make sure your design matches the figma design in the works section. Right now it is nowhere near similar. I know you can do it 👍 
Figma Design             |  Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/60618877/178113440-f43a95f2-172b-4762-b5e6-2071b44e3376.png)  |  ![image](https://user-images.githubusercontent.com/60618877/178113458-f649eb8e-b76a-450a-bfc9-fa6be358c0fb.png)